src/test/java/co/com/api/test/PruebaTest.java

```diff
@@ -0,0 +1,15 @@
+package co.com.api.test;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class PruebaTest {
+
+    @Test
+    public void testAddPass() {
```

Comment:
use `test` like a suffix instead of prefix

Reply:
ok